ironic-deployment/ironic/ironic.yaml

```diff
@@ -85,13 +85,13 @@ spec:
             - mountPath: /shared
               name: ironic-data-volume
         - name: ironic-inspector
-          image: quay.io/metal3-io/ironic-inspector
+          image: quay.io/metal3-io/ironic
```

Comment:
I'll put https://github.com/metal3-io/baremetal-operator/pull/851/files#diff-acbaf87bacacde2d8867e2e1629c604256f8c9ebf6c900248f5d8c1620694fabR91 as well here in the same PR

Reply:
I see the point, but I'd rather keep the two changes separated just for greater clarity
The merge is pretty complex, involves various components, and I think having small changes is better verifiable so if something goes wrong  we have better chance to know what and where